editing_tools/formatter.rb

```diff
@@ -0,0 +1,102 @@
+require 'tempfile'
+require 'pathname'
+require 'fileutils'
+require 'readline'
+require_relative 'rubima-lint'
+
+class RubimaFomatter
+  class BaseFormatter
+    include RubimaLint::Rules
+    
+    def initialize(line_no, line)
+      @line_no = line_no
+      @line = line
+    end
+
+    def apply
+      pre_checked_line = pre_check
+      return @line if @line == pre_checked_line
+
+      puts "#{formated_message}: line [#{@line_no}]:"
```

Comment:
```suggestion
      puts "#{formatted_message}: line [#{@line_no}]:"
```

https://github.com/rubima/magazine.rubyist.net/pull/440#issuecomment-1581582284